From 486c876dd5aae619df217b496206685f7c5d8443 Mon Sep 17 00:00:00 2001
From: Guannan Ren <gren@redhat.com>
Date: Mon, 1 Jul 2013 14:33:06 -0400
Subject: [PATCH 1/3] virtManager: specify missing argument in a call to
 _simple_dialog

(chunk cherry picked from commit d6fc079ae88da2433ae257b25e58f48403b78cf7)

it was mistakenly not included in the previous backport.

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1093980
---
 src/virtManager/error.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/virtManager/error.py b/src/virtManager/error.py
index 7870061..0f14440 100644
--- a/src/virtManager/error.py
+++ b/src/virtManager/error.py
@@ -113,7 +113,7 @@ class vmmErrorDialog(vmmGObject):
         logging.debug("Validation Error: %s" % text1)
         dtype = gtk.MESSAGE_ERROR
         buttons = gtk.BUTTONS_OK
-        self._simple_dialog(dtype, buttons, text1, text2, title, async)
+        self._simple_dialog(dtype, buttons, text1, text2, title, None, async)
         return False
 
     def show_info(self, text1, text2=None, title="", widget=None, async=True):
-- 
1.9.0

